Inseok_45/models/userDao.js

```diff
@@ -20,4 +21,21 @@ const createUser = async ( firstName, lastName, email, phoneNumber, age, userNam
     };
 };
 
-module.exports = { createUser };
\ No newline at end of file
+const getUserById = async ( userName ) => {
```

Comment:
```suggestion
const getUserByName = async ( userName ) => {
```
argument로 userName을 받고 있는데, getUserById는 적합하지 않습니다.